### PR TITLE
feat(UI): Show Episode Runtime

### PR DESCRIPTION
**Pull request by KingLucius**

### CODE REVIEW

File: library/src/commonMain/kotlin/com/lagradost/cloudstream3/MainAPI.kt [hunk 1]
```diff
@@ -1710,7 +1710,25 @@ data class Episode(
     var rating: Int? = null,
     var description: String? = null,
     var date: Long? = null,
-)
+    var runTime: Int? = null,
```
Comment (fire-light42) [inline library/src/commonMain/kotlin/com/lagradost/cloudstream3/MainAPI.kt:1713]: Add a comment about it being in seconds or suffix with Sec

File: app/src/main/java/com/lagradost/cloudstream3/ui/result/EpisodeAdapter.kt [hunk 2]
```diff
@@ -292,6 +297,12 @@ class EpisodeAdapter(
                     episodeDate.isVisible = false
                 }
 
+                episodeRuntime.setText(
+                    txt(
+                        card.runTime?.times(60L)?.let { secondsToReadable(it.toInt(), "0 mins") }
```
Comment (fire-light42) [inline app/src/main/java/com/lagradost/cloudstream3/ui/result/EpisodeAdapter.kt:302]: 0 mins? It should say 1 min or nothing at all.

Comment (KingLucius) [reply app/src/main/java/com/lagradost/cloudstream3/ui/result/EpisodeAdapter.kt:302]: Any chance that this one will every be triggered?

**Review by fire-light42 — APPROVED** [review]
This can be merged anytime, but MainAPI changes might need to be postponed until stable release.

**Comment by KingLucius** [comment]
> This can be merged anytime, but MainAPI changes might need to be postponed until stable release.

There is a backwards compatibility constructor for that

**Comment by Luna712** [comment]
> > This can be merged anytime, but MainAPI changes might need to be postponed until stable release.
> 
> There is a backwards compatibility constructor for that

@KingLucius 

AFAIK that only works for some extensions when there is a stable release as has happened with my last content rating PR which also had a backwards compatible constructor.

**Comment by Luna712** [comment]
Otherwise extensions become incompatible on stable

**Comment by KingLucius** [comment]
> > > This can be merged anytime, but MainAPI changes might need to be postponed until stable release.
> > 
> > There is a backwards compatibility constructor for that
> 
> @KingLucius 
> 
> AFAIK that only works for some extensions when there is a stable release as has happened with my last content rating PR which also had a backwards compatible constructor.

That needs more testing for check what is the issue
Anyway we are not in hurry, we can wait no issue

**Pull request merged**